document-generator-core/swagger-2.0/spec/document-generator.json

```diff
@@ -0,0 +1,57 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Document generator API",
+    "description": "An API for generation of documents",
+    "version": "1.0.0"
+  },
+  "host": "localhost",
+  "schemes": [
+    "http"
+  ],
+  "tags": [
+    {
+    "name": "document-generator"
+    }
+  ],
+  "paths": {
+    "/document-generate": {
+      "post": {
+        "x-operationName": "generate",
+        "summary": "Generation of documents",
+        "description": "Generate a document using the request to obtain the document data and send to render service",
+        "tags": [
+          "document-generator"
+        ],
+        "consumes": [
+          "text/json"
+        ],
+        "parameters": [
+          {
+            "name": "data",
+            "in": "body",
+            "description": "required parameters to generate a document",
+            "required": true,
+            "schema": {
+              "$ref": "http://localhost:3123/swagger-2.0/models/document-generator.json#/definitions/documentGeneratorRequest"
+            }
+          }
+        ],
+        "responses": {
+          "201": {
+            "description": "The document has been created",
+            "schema": {
+              "$ref": "http://localhost:3123/swagger-2.0/models/document-generator.json#/definitions/documentGeneratorResponse"
+            }
+          },
+          "400": {
+            "description": "Unable to handle given request"
+          },
+          "401": {
+            "description": "Not authorized to add the resource "
```

Comment:
doesn't sound right here

Reply:
what doesn't sound right?

Reply:
the description `Not authorized to add the resource` - we're not adding a resource